packages/rum/src/domain/record/serialization/serializeAttribute.ts

```diff
@@ -61,6 +61,11 @@ export function serializeAttribute(
       // Exception: it's safe to reveal the `${PRIVACY_ATTR_NAME}` attr
       return CENSORED_STRING_MARK
     }
+
+    // mask iframe srcdoc
+    if (tagName === 'IFRAME' && (attributeName === 'src' || attributeName === 'srcdoc')) {
```

Comment:
💬 **Thought**: I don't think we need to handle `src` case, since it is not recorded.

Reply:
I removed the src part.